src/App.tsx

```diff
@@ -1,61 +1,39 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { getUserById } from './services/user';
+import todosFromServer from './api/todos';
+
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewTodoId(todos: Todo[]) {
+  return Math.max(...todos.map(todo => todo.id)) + 1;
+}
+
```

Comment:
In my opinion, it would be better if this function was in the file “TodoForm.tsx”, so as not to put a zero id, but yes, you will have to change a little the logic of your array with todo. All this is subjective, so I approve.